internal/pkg/fflag/noop/client.go

```diff
@@ -0,0 +1,11 @@
+package noopfflag
```

Comment:
@Dotunj I don't think we need a noop for this. From my understanding, It's an HTTP wrapper.